File: CS/DXPivotGrid_ChartsIntegration/Internal.cs
using System;
using System.Globalization;
using System.Windows;
using System.Windows.Data;
using System.Windows.Markup;

namespace DXPivotGrid_ChartsIntegration {
    public partial class MainWindow : Window {
        private void Window_Loaded(object sender, RoutedEventArgs e) {
            pivotGridControl1.BestFit();
        }
    }
    public class InverseBooleanConverter : MarkupExtension, IValueConverter {
        #region IValueConverter Members
        public object Convert(object value, Type targetType, 
                object parameter, CultureInfo culture) {
            return !(bool)value;
        }
        public object ConvertBack(object value, Type targetType, 
                object parameter, CultureInfo culture) {
            return !(bool)value;
        }
        #endregion
        public override object ProvideValue(IServiceProvider serviceProvider) {
            return this;
        }
    }
}

File: VB/DXPivotGrid_ChartsIntegration/Internal.vb
Imports System
Imports System.Globalization
Imports System.Windows
Imports System.Windows.Data
Imports System.Windows.Markup

Namespace DXPivotGrid_ChartsIntegration

    Public Partial Class MainWindow
        Inherits Window

        Private Sub Window_Loaded(ByVal sender As Object, ByVal e As RoutedEventArgs)
            Me.pivotGridControl1.BestFit()
        End Sub
    End Class

    Public Class InverseBooleanConverter
        Inherits MarkupExtension
        Implements IValueConverter

'#Region "IValueConverter Members"
        Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.Convert
            Return Not CBool(value)
        End Function

        Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As CultureInfo) As Object Implements IValueConverter.ConvertBack
            Return Not CBool(value)
        End Function

'#End Region
        Public Overrides Function ProvideValue(ByVal serviceProvider As IServiceProvider) As Object
            Return Me
        End Function
    End Class
End Namespace
